multi-agent/lib/mas/elements/nodes/a2a_agent/a2a_agent_node.py

```
@@ -234,6 +231,28 @@ def _build_agent_results_context(self, thread_id: str) -> Optional[ChatMessage]:
+        A2AProvider uses for send_message_sync.
+        """
+        if not self._auth_credential:
+            return
+        try:
+            with get_async_bridge() as bridge:
+                headers = bridge.run(self._auth_credential.get_headers())
+            if headers:
+                self.a2a_provider.update_headers(headers)
+            else:
+                logger.warning("A2AAgentNode[%s]: get_headers() returned empty", self.name)
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)